src/ansible_navigator/runner/command_base.py

```diff
@@ -25,19 +24,20 @@ def __init__(
         wrap_sh: Optional[bool] = False,
         **kwargs,
     ):
-        """Base class to handle common arguments of ``run_command`` interface for ``ansible-runner``
-        Args:
-            executable_cmd ([str]): The command to be invoked.
-            cmdline ([list], optional): A list of arguments to be passed to the executable command.
-                                        Defaults to None.
-            playbook ([str], optional): The playbook file name to run. Defaults to None.
-            inventory ([list], optional): List of path to the inventory files. Defaults to None.
-            wrap_sh ([bool], optional): Wrap the command with `sh -c`, disregarding stderr output.
-                                        Runner will pass --tty in almost all (default) cases. When
-                                        that flag is given, docker and podman will combine the
-                                        process's stdout and stderr into just stdout. Wrapping the
-                                        command with sh allows us to disregard the stderr output
-                                        when we only care about stdout.
+        """Handle common arguments of ``run_command`` interface for ``ansible-runner``.
+
```

Comment:
more of these, we can drop the \, max out the line and indent just 4

Reply:
I think this is fixed... check it out now with the changes.